docs/releases/3.0.1.txt

```diff
@@ -13,3 +13,7 @@ Bugfixes
   inside Jupyter and other environments that force an async context, by adding
   and option to disable :ref:`async-safety` mechanism with
   ``DJANGO_ALLOW_ASYNC_UNSAFE`` environment variable (:ticket:`31056`).
+
+* Fixed a regression in Django 3.0 where ``RegexPattern``, used by
+  :func:`~django.urls.re_path`, returns positional arguments to be passed to
```

Comment:
`would return`?  i.e. It **used** to do this but we fixed it. (It's not current behaviour.)

?

Reply:
I think we use present tenses in release notes ([an example](https://github.com/django/django/blob/master/docs/releases/2.2.2.txt#L40-L59)), i.e. Django 3.0 still behaves that way :thinking:

Reply:
Updated.